src/triggers.js

```diff
@@ -961,7 +951,7 @@ export function getRequestFileObject(triggerType, auth, fileObject, config) {
 }
 
 export async function maybeRunFileTrigger(triggerType, fileObject, config, auth) {
-  const fileTrigger = getFileTrigger(triggerType, config.applicationId);
+  const fileTrigger = getTrigger(FileClassName, triggerType, config.applicationId);
```

Comment:
I think I overlooked it, but what is the `FileClassName`?

Reply:
https://github.com/parse-community/parse-server/blob/48bd512eeb47666967dff8c5e723ddc5b7801daa/src/triggers.js#L24